src/GraphQL/Builders/FieldBuilder.cs

```diff
@@ -210,7 +210,7 @@ private FieldBuilder(FieldType fieldType)
         /// </summary>
         public virtual FieldBuilder<TSourceType, TReturnType> Subscribe(Func<IResolveFieldContext<TSourceType>, IObservable<TReturnType?>> sourceStreamResolver)
```

Comment:
For a whole picture, this method is better to rename too - `ResolveStream` / `ResolveStreamAsync` so to have the whole API set in the same style.

Reply:
Sounds good

Reply:
Done